salvo/src/lib/docker/docker_volume.py

```diff
@@ -0,0 +1,68 @@
+"""
```

Comment:
I may be misreading this, but it looks like we are leaving the original [docker_image.py](https://github.com/abaptiste/envoy-perf/blob/custom_exceptions/salvo/src/lib/docker_image.py) and [docker_volume.py](https://github.com/abaptiste/envoy-perf/blob/custom_exceptions/salvo/src/lib/docker_volume.py) behind.

Was the intention to move them to the new directory? If yes, do you mind updating this to a git move using the `git mv` command which will improve both the code review and the git history we leave behind?
 The simplest way to do that is:

1. Backup content of these two files in your local client.
1. Revert these new files.
1. `git mv` from the original files into this location
1. replace the file content from backups, commit and push.

Git history should take care of the rest and the PR should help the reviewer see what if any changes were performed.

Reply:
Those should be removed.  I did not notice that.  Will clean up.

Reply:
Thank you, this made it much easier to review.